url/a-element.html

```diff
@@ -9,11 +9,11 @@
 var setup = async_test("Loading data…")
 setup.step(function() {
   var request = new XMLHttpRequest()
-  request.open("GET", "urltestdata.txt")
+  request.open("GET", "urltestdata.json")
   request.send()
   request.responseType = "text"
```

Comment:
I wasn't sure it was a good ideal to couple to json responseType support. But I guess all modern browsers have that.